app/models/config.rb

```diff
@@ -335,4 +342,10 @@ def validate_length
       end
       true
     end
+
+    def validate_only_one_practical_support
+      # TODO - Should not be able to use both practical support configs at the
+      # same time.
+      true
+    end
```

Comment:
After this is done, I'm thinking about opening a separate issue to combine "practical support mode" and "hide practical supports" into a single option — practical supports only, on, off. in the interim, should we have some validation? Weird things will happen if we turn on both options.

How would that best work? We can't compare the actual saved values because on validation, the submitted value has not yet been committed. We could have a conditional check based on the option key?

Reply:
Let me say a bit more about the thinking here, and let's see if that makes sense with how you're thinking about it. I think there are three possible states:

* fund is not doing practical support at all (in which case we'd want to hide these options) - unusual but it happens
* fund is doing pledges AND practical support - most of our clientele
* fund is doing zero pledges, ONLY practical support - unusual but it happens

So I think you're def right that 'hide practical support on' and 'practical support only on' doesn't make sense, but I think the other cases (hide off, practical only on; hide on, practical only off;  hide off, practical only off) are cases I think we'd like to support. Does that match your understanding?